File: Java/Program/CPU.java
public class CPU extends KomponenInternal {
    private int jumlahCore;
    private double kecepatanGHz;

    // Constructor
    public CPU() {
        super();
        this.jumlahCore = 0;
        this.kecepatanGHz = 0.0;
    }

    public CPU(String merk, String nama, int harga, String jenisSlot, boolean terpasang, 
               int jumlahCore, double kecepatanGHz) {
        super(merk, nama, harga, jenisSlot, terpasang);
        this.jumlahCore = jumlahCore;
        this.kecepatanGHz = kecepatanGHz;
    }

    // Setter methods
    public void setJumlahCore(int jumlahCore) { this.jumlahCore = jumlahCore; }
    public void setKecepatanGHz(double kecepatanGHz) { this.kecepatanGHz = kecepatanGHz; }

    // Getter methods
    public int getJumlahCore() { return jumlahCore; }
    public double getKecepatanGHz() { return kecepatanGHz; }

    // Tampilkan Info
    @Override
    public String tampilkanInfo() {
        return super.tampilkanInfo() + " | Core: " + jumlahCore + " | Kecepatan: " + kecepatanGHz + " GHz";
    }
}

File: Python/Program/CPU.py
from KomponenInternal import KomponenInternal

class CPU(KomponenInternal):
    def __init__(self, merk="", nama="", harga=0, jenis_slot="", terpasang=False, jumlah_core=0, kecepatan_ghz=0.0):
        super().__init__(merk, nama, harga, jenis_slot, terpasang)
        self.jumlah_core = jumlah_core
        self.kecepatan_ghz = kecepatan_ghz
    
    # Setter
    def set_jumlah_core(self, jumlah_core):
        self.jumlah_core = jumlah_core
    
    def set_kecepatan_ghz(self, kecepatan_ghz):
        self.kecepatan_ghz = kecepatan_ghz
    
    # Getter
    def get_jumlah_core(self):
        return self.jumlah_core
    
    def get_kecepatan_ghz(self):
        return self.kecepatan_ghz
    
    # Tampilkan Info
    def tampilkan_info(self):
        return f"{super().tampilkan_info()} | Core: {self.jumlah_core} | Kecepatan: {self.kecepatan_ghz} GHz"